src/PostgREST/Error.hs

```diff
@@ -784,8 +816,8 @@ buildErrorCode code = case code of
   JWTErrorCode02         -> "PGRST302"
   JWTErrorCode03         -> "PGRST303"
 
-  InternalErrorCode00    -> "PGRSTX00"
+  InternalErrorCode00    -> "PGRSTX00" -- TODO: Ponder - We never really return this error, we kill pgrst early in App.hs
```

Comment:
I think killing PostgREST only happens at startup? I don't think we kill it if it's already running and some unknown Hasql error happens. We've already found an error this way: https://github.com/PostgREST/postgrest/issues/3126

Reply:
Oh, I now understand that it only happens in rare cases indicating a bug, that's why it is not covered by tests.